masq/src/commands/exit_location_command.rs

```diff
@@ -501,30 +499,48 @@ pub mod tests {
         ])
         .unwrap_err();
 
+        let result_expected =
+            "cannot be used with one or more of the other specified arguments\n\nUSAGE:\n";
+        let expected_one = "show-countries";
+        let expected_two = "country-codes";
         assert!(
             result.contains(result_expected),
             "result was {:?}, but expected {:?}",
             result,
             result_expected,
         );
+        assert!(
+            result.contains(expected_one) || result.contains(expected_two),
+            "result was not containing neither of those {:?}, nor {:?}",
```

Comment:
How about, "result should have contained either {:?} or {:?}, but was {:?} instead"?